src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
 - [Implementing new features](./implementing_new_features.md)
 - [Stabilizing Features](./stabilization_guide.md)
 - [Debugging the Compiler](./compiler-debugging.md)
+    - [Debugging Support in Rust Compiler](./debugging-support-in-rustc.md)
```

Comment:
This chapter is about compiler support for debugging _Rust_, not debugging support for debugging the _compiler_, right? In that case it might make sense to clarify that in the first paragraph. It should also probably be its own chapter (perhaps next to the LLVM/codegen chapter?).

Reply:
In contrast, "Debugging the Compiler" is about debugging the compiler itself.

Reply:
Oh yea, good point. IIUC, this is about debugging infrastructure around rustc+tools (gdb, lldb) to debug Rust code. True. I will add the clarification.

Reply:
FYI: I kept this page under "Debugging the Compiler" because of [this comment](https://github.com/rust-lang/rustc-guide/pull/316#issuecomment-489690431) by @igaray. Should I change it?

Reply:
Hmm @igaray thoughts? I think it should be moved...

Reply:
Given that there is some confusion, then I think it's ok to have it in it's own section. I still think they are somewhat related topics though, so maybe two chapters in close proximity?
In any case we know that further down the line a reorganization of the book is due so I don't think it matters much right now.

Reply:
Personally, I don’t think they are very related, but I agree with you that it doesn’t matter too much.

Reply:
Sorry to have triggered bike-shedding. I also agree it does not matter as much right this second.